user-story-nfdi4ing-instrument/functions.py

```diff
@@ -53,27 +54,38 @@ def get_metadata_display(doi):
 
     return data
 
-def format_citations(events, include_authors=False):
+def format_citations(events, doi, include_authors=False):
```

Comment:
How would DOI input be used here?

Reply:
That was left over from some earlier testing from the extract_ids() method, I can go ahead and remove that